src/unibasicTemplates/getServerInfo.njk

```diff
@@ -4,12 +4,12 @@ subroutine mvom_getServerInfo(options, output)
 
 {% include "./partials/errorConstants.njk" %}
 
-if udoSetProperty(output, 'date', date()) then
+if udoSetProperty(output, 'date', date(), UDO_NUMBER) then
```

Comment:
These values were typed as number for the subroutine output, but the db server was actually returning strings.  This didn't seem to cause any issues, but this change makes the output consistent with expectations.